meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-mender/mender-client/mender-client_%.bbappend

```diff
@@ -1,21 +1,21 @@
 EXTRADEPS = ""
 EXTRADEPS:tegra = "tegra-bup-payload tegra-boot-tools tegra-boot-tools-nvbootctrl tegra-boot-tools-lateboot${@' libubootenv-fake' if d.getVar('PREFERRED_PROVIDER_virtual/bootloader').startswith('cboot') else ''}"
 EXTRADEPS:tegra210 = "tegra-bup-payload tegra-boot-tools"
```

Comment:
I think we can drop the tegra210 reference here since Nano won't be supported on this branch.

Reply:
@dwalkes I cleaned up this file to remove tegra210 as you suggested and tested AGX Xavier and AGX Orin, both work